src/main/resources/db/migration/V6.1.0.1__9686-move-harvestingclient-id.sql

```diff
@@ -1,5 +1,7 @@
 ALTER TABLE dvobject ADD COLUMN IF NOT EXISTS harvestingclient_id BIGINT;
 
+ALTER TABLE dvobject ADD CONSTRAINT fk_dvobject_harvestingclient_id FOREIGN KEY (harvestingclient_id) REFERENCES harvestingclient (id);
```

Comment:
FWIW: We usually try to make Flyway scripts idempotent, and usually try to avoid changing scripts that have gone into develop. For relatively new ones, we've sometimes made changes and then just send a note to the dev mailer (that if you've deployed develop already, you'll have to delete this flyway entry if you deploy again after the PR).

To make this idempotent - look at the past scripts using 'CONSTRAINT' - they either drop the constraint first if it exists, or they catch the error if adding it is a duplicate.